File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/city/ResultOpenAQCity.java

package test.platzi.com.raian.com.org.simpledynamit.model.city;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ResultOpenAQCity {

    @SerializedName("meta")
    @Expose
    private Meta meta;
    @SerializedName("results")
    @Expose
    private List<Result> results = null;

    public Meta getMeta() {
        return meta;
    }

    public void setMeta(Meta meta) {
        this.meta = meta;
    }

    public List<Result> getResults() {
        return results;
    }

    public void setResults(List<Result> results) {
        this.results = results;
    }

}

File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/city/ResultOpenAQCity.kt
package test.platzi.com.raian.com.org.simpledynamit.model.city

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResultOpenAQCity {

    @SerializedName("meta")
    @Expose
    var meta: Meta? = null
    @SerializedName("results")
    @Expose
    var results: List<Result>? = null

}
